src/bootstrap/test.rs

```diff
@@ -599,7 +599,6 @@ impl Step for RustdocTheme {
             .env("RUSTDOC_LIBDIR", builder.sysroot_libdir(self.compiler, self.compiler.host))
             .env("CFG_RELEASE_CHANNEL", &builder.config.channel)
             .env("RUSTDOC_REAL", builder.rustdoc(self.compiler))
-            .env("RUSTDOC_CRATE_VERSION", builder.rust_version())
```

Comment:
Hm I guess we're saying this is fine to drop because this is just for tests?

Reply:
Yea, I'm not sure why it was added in the first place (#47761). The rustdoc wrapper at that time looked like [this](https://github.com/rust-lang/rust/blob/51580d46f919c1f97d82aeca1ea1086c545c7484/src/bootstrap/bin/rustdoc.rs#L61-L71), so maybe it was relying on adding `-Zunstable-options`? Not sure.  Nothing in the rustdoc-themes test appears to care about the version.

Similarly, the usages of `rustdoc_cmd` do not need it, as they are only used for the [`Standalone` step](https://github.com/rust-lang/rust/blob/8e21bd0633b8d970646ee6eb706c9e8acfad19af/src/bootstrap/doc.rs#L350) which doesn't use the crate version (it doesn't build crate docs) and the [markdown tests](https://github.com/rust-lang/rust/blob/8e21bd0633b8d970646ee6eb706c9e8acfad19af/src/bootstrap/test.rs#L1506) and the [TRPL redirects](https://github.com/rust-lang/rust/blob/8e21bd0633b8d970646ee6eb706c9e8acfad19af/src/bootstrap/doc.rs#L255).